frontend/src/routes/(authenticated)/myaccount/+page.ts

```diff
@@ -0,0 +1,42 @@
+import type { PageLoad } from './$types';
+import type {
+  $OpResult,
+  ChangeUserAccountDataMutation,
+  ChangeUserAccountDataInput,
+
+} from '$lib/gql/types';
+import { getClient, graphql } from '$lib/gql';
+import { invalidate } from '$app/navigation';
+
+export async function _changeUserAccountData(input: ChangeUserAccountDataInput): $OpResult<ChangeUserAccountDataMutation> {
+  //language=GraphQL
+  const result = await getClient()
+    .mutation(
+      graphql(`
+        mutation ChangeUserAccountData($input: ChangeUserAccountDataInput!) {
+          changeUserAccountData(input: $input) {
+            user {
+              id
+              name
+              username
+              email
+            }
+            errors {
+              ... on Error {
+                message
+              }
+            }
+          }
+        }
+      `),
+      { input: input },
+      //invalidates the graphql user cache, but who knows
+      { additionalTypenames: ['Users'] },
+    )
+    .toPromise();
+  if (!result.error) void invalidate(`user:${input.userId}`);
+  return result;
+}
+export const load = (async () => {
```

Comment:
I believe this is unnecessary, though I haven't been able to get the PR branch to compile due to the GraphQL autogeneration issue we're having with the admin page right now. So I can't check whether Svelte-Kit complains if you remove the load function. I believe it doesn't, but if you remove the load function here and Svelte-Kit complains then you can put it back.

Reply:
Thanks! I'll try without it.

Reply:
It seems to work either way